google-cloud-build/ci-scripts/scripts/generate_flavor_options.sh

```diff
@@ -6,7 +8,7 @@ function generate_flavor_options(){
   do
     FLAVOR_OPTIONS="$FLAVOR_OPTIONS --flavor-path 'flavors/$FLAVOR'"
   done
-  if [ -z "$FLAVOR_OPTIONS=" ]
+  if [ -z "$FLAVOR_OPTIONS" ]
```

Comment:
Was a bug?

Reply:
probably, it looks like it

Reply:
but is it actually used, is the other question

Reply:
I think, the file is unused, but please check again.

Reply:
Yes, the file is sourced in the push.sh and export.sh google build ci scripts.

Reply:
Ok, I think this explains why this bug didn't cause any problems, because the push and export specify the flavor always